test-apps/display-test-app/package.json

```diff
@@ -93,8 +93,8 @@
     "eslint-plugin-deprecation": "1.2.1",
     "eslint-plugin-import": "2.23.4",
     "eslint-plugin-prefer-arrow": "1.2.3",
-    "eslint-plugin-react": "7.24.0",
-    "eslint-plugin-react-hooks": "4.2.0",
+    "eslint-plugin-react": "^7.32.2",
```

Comment:
honestly should be able to delete all eslint related deps in this test app. should be able to just use the itwin/eslint-plugin itself

Reply:
Okay, I will try that

Reply:
hmm you might have to leave eslint in
<img width="1014" alt="image" src="https://user-images.githubusercontent.com/11051042/224054573-8f32bc12-b49c-4f45-bc8b-8fec1f733c16.png">
its listed as a peer dep
<img width="498" alt="image" src="https://user-images.githubusercontent.com/11051042/224054619-6d00537f-c402-4804-bd3d-3a1191211a76.png">

Reply:
Okay, I'm surprised why `rush lint` didn't show any errors.